pkg/cmd/cli/secrets/dockercfg.go

```diff
@@ -37,9 +37,9 @@ If you don't already have a .dockercfg file, you can create a dockercfg secret d
 
   $ %s SECRET_NAME --docker-server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL
 
-If you do already have a .dockercfg file, you can create a dockercfg secret by using:
+If you do already have a .dockercfg file, you can create a dockercfg secret by using the 'oc secrets new' command:
 
-  $ %s SECRET_NAME path/to/.dockercfg
+  $ %[2]s SECRET_NAME path/to/.dockercfg
```

Comment:
oh, didn't notice we were already passing in a second arg for the new secret command. in that case, switch all the `%s` in the format string to `%[n]s` style